pkgs/development/python-modules/webssh/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, tornado, paramiko }:
+
+buildPythonPackage rec {
+  pname = "webssh";
+  version = "1.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Au6PE8jYm8LkEp0B1ymW//ZkrkcV0BauwufQmrHLEU4=";
+  };
+
+  doCheck = false;
```

Comment:
There seem to be tests, why disable them?

Reply:
Fixed - when I wrote this package with an earlier version, the tests failed (if I remember correctly, something to do with it trying to do network connections), but seems to be working now.